onnxruntime/core/providers/dml/DmlExecutionProvider/src/ApiTraits.cpp

```diff
@@ -241,6 +241,7 @@ DML_OPERATOR_TYPE ApiTraits::StringifyHelpers::FromString(std::string_view value
         {"DML_OPERATOR_ACTIVATION_SWISH", DML_OPERATOR_ACTIVATION_SWISH},
         {"DML_OPERATOR_ACTIVATION_HARD_SWISH", DML_OPERATOR_ACTIVATION_HARD_SWISH},
         {"DML_OPERATOR_RESAMPLE2", DML_OPERATOR_RESAMPLE2},
+        {"DML_OPERATOR_RESAMPLE3", DML_OPERATOR_RESAMPLE3},
```

Comment:
We should also have entries for Fold and Unfold.

Reply:
Interesting. I see them below, but not here:

```
template <>
struct OperatorDescTraits<DML_FOLD_OPERATOR_DESC>
{
    static constexpr DML_OPERATOR_TYPE Type = DML_OPERATOR_FOLD;
};

template <>
struct OperatorDescTraits<DML_UNFOLD_OPERATOR_DESC>
{
    static constexpr DML_OPERATOR_TYPE Type = DML_OPERATOR_UNFOLD;
};
```

Reply:
Yeah, it looks like there was another entry in the schema that was missing. It shouldnt cause a breaking change though.
Lets keep ApiTraits in sync and update with the following release of DML which has the fix.